cli/rocks/rocks.go

```diff
@@ -27,6 +27,7 @@ const (
 	rocksRepoManifestName     = "manifest"
 	repoRocksPathEnvVarName   = "TT_CLI_REPO_ROCKS"
 	tarantoolPrefixEnvVarName = "TT_CLI_TARANTOOL_PREFIX"
+	tarantoolDefaultPrefixDir = "/usr"
```

Comment:
Changes related to the `rocks` module are not related to the addition of tarantool 3.0 to CI and can be moved to a separate preparatory commit.

Reply:
Done.